### PR TITLE
Add lag compensation to many prob calls in reagent `on_mob_life()`s.

### PR DESCRIPTION
**Pull request by Tarmunora**
[BALANCE] [FIX]
## About the PR 
Title
Probs that lead into a value already lag-compensated were generally not changed, as the math works out better that way

Also added lag compensation to some other parts of chemicals that I saw were missing as I was scrolling through prob calls

## Why's this needed? 
Some effects weren't happening as often as they should be (notable freeze/dbreath)

### CODE REVIEW

File: _std/macros/reagents.dm [hunk 1]
```diff
@@ -0,0 +1 @@
+#define probmult(x) (prob(percentmult((x), mult))) //assumes lag-compensation multiplier is always called mult
```
Comment (ZeWaka) [inline _std/macros/reagents.dm:1]: could take in the lag-compensation multiplier as an arg

Comment (pali6) [reply _std/macros/reagents.dm:1]: Unless I missed something very obvious optional arguments aren't a thing with macros. However, that can be fixed by clever use of a few macros + varargs probably.

File: _std/__std.dme [hunk 2]
```diff
@@ -74,6 +74,7 @@
 #include "macros\movement_modifiers.dm"
 #include "macros\plane.dm"
 #include "macros\robot.dm"
+#include "macros\reagents.dm"
```
Comment (pali6) [inline _std/__std.dme:77]: Wouldn't using chemistry.dm work?

**Review by pali6 — COMMENTED** [review]
Looks good to me, though I half expect this to subtly break something.

File: _std/__std.dme (hunk 2 above)
Comment (Tarmunora) [reply _std/__std.dme:77]: Probably?

File: _std/macros/reagents.dm (hunk 1 above)
Comment (Tarmunora) [reply _std/macros/reagents.dm]: Probably could, but maybe not sanely - using the `percentmult(x, mult)` macro in a prob call is probably better practice anyways, but this was easier 🤷

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**